core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2362,6 +2362,12 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_LOST_WORKER_FILE_DETECTION_ENABLED =
+      booleanBuilder(Name.MASTER_LOST_WORKER_FILE_DETECTION_ENABLED)
+          .setDefaultValue(false)
+          .setDescription("Enable lost worker file detector.")
+          .setScope(Scope.SERVER)
+          .build();
```

Comment:
Per our discussion, this config property is not needed, let's remove it.

Reply:
done